src/tool_operate.c

```diff
@@ -1473,6 +1473,7 @@ static CURLcode operate_do(struct GlobalConfig *global,
         /* new in 7.25.0 and 7.44.0 */
         {
           long mask = (config->ssl_allow_beast ? CURLSSLOPT_ALLOW_BEAST : 0) |
+		      (config->ssl_disable_tls13_middlebox ? CURLSSLOPT_TLS13_MIDDLEBOX: 0) |
```

Comment:
Indention is off by two spaces to the left here.

Reply:
Indentation fixed